packages/meditrak-app/app/sync/syncMigrations.js

```diff
@@ -38,9 +38,9 @@ const migrations = {
   7: async (synchroniser, setProgressMessage) => {
     await resyncRecordTypes(synchroniser, setProgressMessage, ['survey_screen_component']);
   },
-  // Resync all entities so that the new attributes column come through
+  // Resync all entities and options so that the new attributes column come through
   8: async (synchroniser, setProgressMessage) => {
-    await resyncRecordTypes(synchroniser, setProgressMessage, ['entity']);
+    await resyncRecordTypes(synchroniser, setProgressMessage, ['entity', 'option']);
```

Comment:
I just slipped this in (not related to react-navigation upgrade) because I just realised beside entities, option also has the new `attributes` column which need to be resynced all the way